clang/lib/Sema/SemaChecking.cpp

```diff
@@ -10808,7 +10808,7 @@ static std::optional<IntRange> TryGetExprRange(ASTContext &C, const Expr *E,
       // sign bit. If the range was not non-negative, we need an extra bit
```

Comment:
```suggestion
      // sign bit. Otherwise, we need an extra bit
```

Seems like it reads better. Should we also mention in the comment we are capping by MaxWidth?